ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/DeploymentsTable.tsx

```diff
@@ -124,6 +128,12 @@ function DeploymentsTable({
                                         <Truncate position="middle" content={name} />
                                     </Link>
                                 </Td>
+                                <Td
+                                    dataLabel="Risk priority"
+                                    className="pf-v5-u-pr-2xl pf-v5-u-text-align-center-on-md"
```

Comment:
This layout does the following:
- Centers the number in the column cell
- Added enough padding on the righthand side to account for the sorting icon in the column header

Since this is the default sort, and the primary method of prioritization, I wanted to make sure the value was displayed clearly.

Neither the default layout nor right-aligned were very readable:
![image](https://github.com/stackrox/stackrox/assets/1292638/c064132f-11d8-4168-a8aa-9323919ff31c)
![image](https://github.com/stackrox/stackrox/assets/1292638/ef35e4b6-a097-44e0-8b55-61ff7d2d0dec)

And centering without the extra padding looked sloppy due to the offset with the header text:
![image](https://github.com/stackrox/stackrox/assets/1292638/a6a8920c-18eb-4d56-8683-adde20a08350)

The easiest to parse for me was the final layout, centered in the cell relative to the header text:
![image](https://github.com/stackrox/stackrox/assets/1292638/956cea13-92c6-443b-bd7b-9f759931399e)

Reply:
Also I would have preferred to set `paddingRight` to the CSS variable values that make up the size of the icon, but it is partially due to an inline style of `1rem`. I figured a `pr-2xl` class wasn't really less maintainable than a css `calc` property in this case.

Reply:
Thank you for exploring possibilities.

What do you think about pasting a link to your comment above into a PatternFly channel? Whether or not there is an alternative, it seems worthwhile for awareness.

Reply:
Hmm so I looked at this with fresh eyes, and it looks like the _technically correct_ spacing is achievable via a mix of PF CSS variables, and is two pixels less than the padding class I added here:
```diff

+ style={{
+  paddingRight:
+    'calc(var(--pf-v5-c-table--cell--PaddingRight) + var(--pf-v5-c-table__sort__button--PaddingRight) + var(--pf-v5-c-table__sort-indicator--MarginLeft) + 1em)',
+  }}
+ className="pf-v5-u-text-align-center-on-md"
- className="pf-v5-u-pr-2xl pf-v5-u-text-align-center-on-md"
```

Any opinion on the tradeoff of readability versus exact correctness and (theoretical) maintainability of staying in sync with PF CSS values?

My gut reaction is that the likelihood of PatternFly changing header spacing variables or `pf-v5-u-pr-2xl` is equally as likely as them removing a CSS variable entirely in a new release (which happened in PF4 -> PF5 in at least one case). I'd favor two pixels of incorrectness here to keep the code more readable for our team.